tasks/bloom-etl/index.ts

```diff
@@ -0,0 +1,49 @@
+import { Runner, Extractor, Transformer, Loader, defaultMap } from "./src/etl"
+import { DbConfig, Jurisdiction, UrlInfo } from "./src/types"
+import { knex } from "knex"
+import axios from "axios"
+
+// No need to retrieve these dynamically since they never change, per Exygy
+const jurisdictions: Array<Jurisdiction> = [
+  {
+    id: "3d0a7a32-345e-4ca6-86d8-66e51d6082db",
+    name: "San Jose",
+  },
+  {
+    id: "3328e8df-e064-4d9c-99cc-467ba43dd2de",
+    name: "San Mateo",
+  },
+  {
+    id: "bab6cb4f-7a5a-4ee5-b327-0c2508807780",
+    name: "Alameda",
+  },
+]
+
+// This is also unlikely to change during the lifetime of this task
+const url: UrlInfo = {
+  base: process.env.EXTERNAL_ENDPOINT_URL || "https://proxy.housingbayarea.org",
```

Comment:
should we be consistent with the other services and put (all of) these into a .env(.template) file?

Reply:
Yeah, I can add that in no problem

Reply:
Let me check with Morgan on the best endpoint URL to use for non-prod.

Reply:
I removed the default value and put the dev endpoint in the .env.template file